app/build.gradle

```diff
@@ -264,6 +269,9 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'androidx.multidex:multidex:2.0.0'
     implementation 'com.google.guava:guava:20.0'
+
+    implementation 'org.apache.commons:commons-lang3:1.3.2'
+    implementation 'com.github.ybq:Android-SpinKit:1.4.0'
```

Comment:
are you using this added libs?

Reply:
I had upgraded client-core and simprints libraries and these were missing. So I checked on CHW and found they were included in the build file and include them.